draft-ietf-oauth-transaction-tokens.md

```diff
@@ -497,17 +507,28 @@ A requester MAY use a self-signed JWT as a `subject_token` value. In that case,
 
 The self-signed JWT MAY contain other claims.
 
+### Unsigned JSON Object Subject Token Type {#unsigned-json-subject-token-type}
+A requester MAY use an unsigned JSON object as a `subject_token` value. In that case, the requester MUST set the `subject_token_type` value to: `urn:ietf:params:oauth:token-type:unsigned_json`. The value of the `subject_token` field MUST be the BASE64URL encoded value of the JSON object as described in {{Section 5 of RFC6848}}.  The JSON object in the subject token MUST contain the following fields:
+
+* `sub`: The subject for whom the Txn-Token is being requested. The Txn-Token Service SHALL use this value in determining the `sub` value in the Txn-Token issued in the response to this request.
+* `exp`: The expiration time of the unsigned JSON object,  which the TTS MAY use as input to determining the lifetime of the Txn-token.
+
+The unsigned JSON object MAY contain other fields, and the Txn-Token Service MAY consider them when generating the Txn-Token.
+
 ## Txn-Token Request Processing
 When the Transaction Token Service receives a Txn-Token Request it MUST validate the requesting workload client authentication and determine if that workload is authorized to obtain the Txn-Tokens with the requested values. The authorization policy for determining such issuance is out of scope for this specification.
 
 Next, the Transaction Token Service MUST validate the `subject_token` and determine the value to specify as the `sub` of the issued Txn-Token. The Txn-Token Service MUST ensure the `sub` value is unique within the trust domain defined by the `aud` claim.
 
 The Transaction Token Service MUST set the `iat` claim to the time of issuance of the Txn-Token.
+
 The Transaction Token Service MUST set the `aud` claim to an identifier representing the Trust Domain of the Transaction Token Service. If the Transaction Token Service supports multiple trust domains, then it MUST determine the correct `aud` value for this request.
-The Transaction Token Service MUST set the `exp` claim to the expiry time of the Txn-Token.
+
+The Transaction Token Service MUST set the `exp` claim to the expiry time of the Txn-Token. The Txn-Token Service MAY consider any `exp` value present in the `subject_token` field of the Txn-Token Request in determining the `exp` value of the resulting Txn-Token.
```

Comment:
Maybe change "`subject_token` field" to "`subject_token` parameter" ?

Reply:
done.